src/index.html

```diff
@@ -11,7 +11,147 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post">
+      <fieldset class="input__container">
+        <legend>Personal information</legend>
+        <label class="input__container-item">
+          Surname:
+          <input
+            type="text"
+            name="surname"
+            autocomplete="off"
+            required
+          >
+        </label>
+        <label class="input__container-item">
+          Name:
+          <input
+            type="text"
+            name="username"
+            autocomplete="off"
+            required
+          >
+        </label>
+        <label class="input__container-item">
+          How old are You?
+          <input
+            type="number"
+            min="0"
+            max="100"
+            name="age"
+            autocomplete="off"
+            value="12"
+          >
+        </label>
+        <label class="input__container-item">
+          Full date of birth:
+          <input
+            type="date"
+            name="date-birth"
+            autocomplete="off"
+          >
+        </label>
+        <label class="input__container-item">
+          I accept the term of the agreement
+          <input
+            type="checkbox"
+            name="agreement"
+            value="yes"
+            autocomplete="off"
+            required
+          >
+        </label>
+      </fieldset>
+      <fieldset class="input__container">
+        <legend>Registration</legend>
+        <label class="input__container-item">
+          E-mail:
+          <input
+            type="email"
+            name="email"
+            placeholder="email@example.com"
+            required
+          >
+        </label>
+        <label class="input__container-item">
+          Password:
+          <input
+            type="password"
+            name="password"
+            minlength="8"
+            maxlength="25"
+            required
+          >
+        </label>
+      </fieldset>
+      <fieldset class="input__container">
+        <legend>An intersting fact about you!</legend>
+        <div class="input__container-item">
+          Do you love cats?
+          <label>
+            <input
+              type="radio"
+              name="animal"
+              value="Yes"
+            >Yes
+          </label>
```

Comment:
```suggestion
          <label>
            <input
              type="radio"
              name="animal"
              value="Yes"
            >

            Yes
          </label>
```
fix the same below

Reply:
[checklist](https://github.com/mate-academy/layout_html-form/blob/master/checklist.md) -> codestyle